pyspecProcScripts/load_data.py

```diff
@@ -38,8 +38,9 @@ def proc_bruker_deut_IR_withecho_mancyc(s, fl=None):
         fl.next("IR prior to FTing ph")
         fl.image(s.C.setaxis("indirect", "#").set_units("indirect", "scan #"))
     s.ft(
-        ["ph1", "ph2"], unitary=True
+        ["ph1", "ph2"]
```

Comment:
can you explain in a comment maybe the first time that you are not doing unitary in order to match the amplitude of the coherence domain signal to its size in each transient

Reply:
added in def prc_spincore_SE_v1 (first popularly used processing in the file)

Reply:
I don't see that here, and this is the first diff in the PR.

Reply:
I said it was in the most popularly used ppg but moved it here so you can see it

Reply:
add the main block of the ppg to the docstring of this function

Reply:
please say here that you are not doing unitary in order to match the amplitude of the coherence domain signal to its size in each transient

Reply:
commented in spincore SE v1 since that is now the first function in the script